Formula/libressl.rb

```diff
@@ -2,9 +2,8 @@ class Libressl < Formula
   desc "Version of the SSL/TLS protocol forked from OpenSSL"
   homepage "https://www.libressl.org/"
   # Please ensure when updating version the release is from stable branch.
```

Comment:
According to the homepage, 3.0.0 is the development release. The stable release is still 2.9.2.

Reply:
Yep, good catch. I just pulled it from the same OpenBSD directory as the stable releases and made a bad assumption.

[https://ftp.openbsd.org/pub/OpenBSD/LibreSSL/](url)

But, checking release notes, they annotated at the beginning of them that this was the first release of the development branch.

[https://ftp.openbsd.org/pub/OpenBSD/LibreSSL/libressl-3.0.0-relnotes.txt](url)

My fault and I'll close this one.